### ISSUE TITLE
Incompatible with `androidx.test:runner:1.5.x`

### ISSUE THREAD

**Issue by Paul-McNeice** [stepstone-tech/AndroidTestXRunner#10]
The `addTests(Description)` method no longer exists on the `TestEventClient` so running tests results in a `NoSuchMethodException`.

I've tried to fix this myself by using reflection to set the `tests` property of `RunnerArgs` to include the "duplicate" tests but this results in the test only being ran once so I'm obviously doing something wrong 😞 

Any update, help or pointers in the right direction would be much appreciated 👍

**Comment by zawadz88** [comment]
Yeah, I had a go at re-enabling support with Runner 1.5.x but sadly to no avail 😬
I found a workaround that means this library is no longer needed though...

Since Parameterized tests are supported since  Runner 1.3.x (https://github.com/android/android-test/releases/tag/androidx-test-1.3.0-beta02), you can actually do sth like this in your tests:

```kotlin
@RunWith(Parameterized::class)
abstract class YourTest {

    companion object {
        @JvmStatic
        @Parameterized.Parameters(name = "{index}")
        internal fun data(): Collection<Int> =
            List(InstrumentationRegistry.getArguments().InstrumentationRegistry.getArguments()
                    .getString("testCount")?.toIntOrNull() ?: 1) { index -> index }
    }

    @Parameterized.Parameter
    @JvmField
    var testSequenceNumber: Int = 0
    
    // your test code here
    }
```
and if you need to execute more than 1 test, provide the `testCount` argument e.g. via:
```
./gradlew connectedDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.testCount=10
```

**Comment by Paul-McNeice** [comment]
Interesting approach, thank you - I'll give it a go :)
